content/files/Akeneo PIM API.postman_collection.json

```diff
@@ -2964,26 +2964,26 @@
 			"name": "Authentification by password",
 			"event": [
 				{
-					"listen": "test",
+					"listen": "prerequest",
```

Comment:
are you sure of that? did you modify something ?

Reply:
Nope, the version of Postman export changed.